src/text-buffer.coffee

```diff
@@ -576,6 +576,7 @@ class TextBuffer
   #   * `getPath` A {Function} that returns the {String} path to the file.
   #   * `createReadStream` A {Function} that returns a `Readable` stream
   #     that can be used to load the file's content.
+  #   * `existsSync` A {function} that returns the {Boolean}, true if the file exists, false otherwise
```

Comment:
Can this be moved below `createWriteStream` so it doesn't split the two `create___Stream` properties up?

Reply:
Great catch, thanks! I've reordered them, such that they write streams are together.